docs/REST.md

```diff
@@ -158,9 +156,11 @@ Body:
 * `manufacturer`: The new manufacturer of the model
 * `vendor`: The bew vendor who sold the model
 * `location`: New location where the model is stored
-* `isFaulty`: Whether the model is faulty of not
-* `faultDescription`: New description of the fault
+* `allowCheckout`: If true, the model can be checked out as if it were an item
 * `price`: New price of one model
+* `count`: The total amount of the model
+* `changeStock`: If true, the number of models in stock is manually changed
```

Comment:
I ran into a problem when it comes to editing a nonserialized model.  Tech support should be able to change the total number (count) of that model freely.  There are three common use cases where this occurs:

1. Tech support receives more of a model.  Perhaps 50 new transistors come in and they need to be added.  Tech support can increase the count by 50 and all is fine.  There are now 50 more models total, as well as 50 more models in stock.  Therefore, the inStock value should automatically be increased by the same amount.
2. Tech support retires one or more of a model in stock.  A few resistors are broken, so they must be removed from the system.  Tech support has the resistors in their possession and simply disposes of them and edits the model to reflect this.  If they dispose of 10 resistors, they decrease the count by 10, decreasing the total, but the number in stock must also be decreased by 10, once again making it make the most sense to automatically remove the same number of models from inStock as from count.
3. Tech support retires a model that is checked out/overdue or has gone missing and is not currently in stock.  Tech support counts the total number of cables at the end of the quarter and realizes they are 5 short.  They have no reason to believe that they will be getting these 5 back, so they edit the total to be 5 less.  However, the number in stock has not changed.  Here it makes more sense to allow the count and inStock values to be changed separately.

I decided to give the form a checkbox which specifies whether or not the stock is changed independently from the total amount of a model.  If left unchecked, changeStock will be false (which makes the name a bit misleading, I suppose) so the stock will be changed automatically by the same amount as the count.  If checked, the form allows the worker to update the inStock value independently, accounting for strange scenarios such as missing or long overdue models.